docs/querying/api.md

```diff
@@ -381,3 +381,61 @@ $ curl http://localhost:9090/api/v1/alertmanagers
   }
 }
 ```
+
+
+## TSDB Admin APIs
+These are APIs that expose database functionalities for the advanced user. These APIs are not enabled unless the `--web.enable-admin-api` is set.
+
+We also expose a gRPC API whose definition can be found [here](https://github.com/prometheus/prometheus/blob/master/prompb/rpc.proto). This is experimental and might change in the future.
+
+### Snapshot
+Snapshot creates a snapshot of all current data into `snapshots/<datetime>-<rand>` under the TSDB's data directory and returns the directory as response.
+
+```
+POST /api/v1/admin/tsdb/snapshot
+```
+
+```json
+$ curl -XPOST http://localhost:9090/api/v1/admin/tsdb/snapshot
+{
+  "name": "2017-11-30T15:31:59Z-2366f0a55106d6e1"
+}
+```
+
+The snapshot now exists at `<data-dir>/snapshots/2017-11-30T15:31:59Z-2366f0a55106d6e1`
+
+
+### Delete Series
+DeleteSeries deletes data for a selection of series in a time range. The actual data still exists on disk and is cleaned up in future compactions or can be explicitly cleaned up by hitting the Clean Tombstones endpoint.
+
+```
+DELETE /api/v1/admin/tsdb/delete_series
+```
+
+URL query parameters:
+
+- `match[]=<series_selector>`: Repeated label matcher argument that selects the series to delete. At least one `match[]` argument must be provided.
+- `start=<rfc3339 | unix_timestamp>`: Start timestamp.
+- `end=<rfc3339 | unix_timestamp>`: End timestamp.
+
+Example:
+
+```json
+$ curl -X DELETE \                                                              
+  -g 'http://localhost:9090/api/v1/series?match[]=up&match[]=process_start_time_seconds{job="prometheus"}'
+```
```

Comment:
The return value should be documented, and below. It should really follow the response docs up the top of this page.

Reply:
We just return a  204 and I documented that.